README.md

```diff
@@ -83,6 +90,8 @@ A dbt profile can be configured to run against AWS Athena using the following co
 | aws_profile_name      | Profile to use from your AWS shared credentials file.                          | Optional  | `my-profile`                               |
 | work_group            | Identifier of Athena workgroup                                                 | Optional  | `my-custom-workgroup`                      |
 | num_retries           | Number of times to retry a failing query                                       | Optional  | `3`                                        |
+| lf_tags               | Default lf tags to apply to any database created by dbt                        | Optional  | `{"origin": "dbt", "team": "analytics"}`   |
```

Comment:
I am not sure to get why lf_tags is now is this PR ? Is it because of rebase ?

Reply:
I have updated the README. Yeah I did an interactive rebase that went horribly wrong. If we could squash the commits in this PR before merging that would be great.